### PR TITLE
Properly advertise the client requests.

### PR DESCRIPTION
**Pull request by cheftako**
Need to work out how to better handle the version information.

### Change description

Properly advertise the client requests.

### Tests you have done

- [x] Run `make ready-pr` to ensure this PR is ready for review.
- [ ] Perform necessary E2E testing for changed resources.

### CODE REVIEW

File: third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/fwtransport/framework_utils.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 const uaEnvVar = "TF_APPEND_USER_AGENT"
 
 func CompileUserAgentString(ctx context.Context, name, tfVersion, provVersion string) string {
-	ua := fmt.Sprintf("Terraform/%s (+https://www.terraform.io) Terraform-Plugin-SDK/%s %s/%s", tfVersion, "terraform-plugin-framework", name, provVersion)
+	ua := fmt.Sprintf("kcc/1.117.0 (+https://github.com/GoogleCloudPlatform/k8s-config-connector) kcc/controller-manager/%s", provVersion)
```
Comment (justinsb) [inline third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/fwtransport/framework_utils.go:22]: Maybe `//go:embed` would be the easiest approach to the version problem?  I think we can `go:embed` just a string.

Comment (cheftako) [reply third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/fwtransport/framework_utils.go]: Done

**Comment by justinsb** [comment]
Thanks @cheftako 

/approve
/lgtm

**Pull request merged**